web/core/components/command-palette/command-modal.tsx

```diff
@@ -46,7 +46,7 @@ export const CommandModal: React.FC = observer(() => {
   // hooks
   const { getProjectById, workspaceProjectIds } = useProject();
   const { isMobile } = usePlatformOS();
-  const { canPerformWorkspaceCreateActions } = useUser();
+  const { getProjectsWithPermissions } = useUser();
```

Comment:
Lets add a member variable canPerformProjectWriteActions or hasCreateAccessToAtleastOneProject.

It will be great if you can find a better name.